run/CESM/HEMCO_Config.rc

```diff
@@ -1292,287 +1367,295 @@ VerboseOnCores:              root       # Accepted values: root all
 # --- CEDS v2 ---
 #
```

Comment:
Heads up: In 14.4.0 there will be further updates for CEDSv2 (diurnal and day-of-week scaling), as well as for 0.1 x 0.1 CEDS emissions.  This section will have to be added for CESM post 14.4.0 release.

Reply:
Noted. Once we release 14.3.1 and merge it into 14.4.0 dev I can update the config file.